core/src/main/scala/cats/syntax/all.scala

```diff
@@ -43,6 +43,7 @@ trait AllSyntax
     with SemigroupKSyntax
     with ShowSyntax
     with StrongSyntax
+    with TrySyntax
```

Comment:
Unfortunately this isn't binary compatible scala 2.11-. 
We can still include these syntax, but this TrySyntax won't be imported using `cats.implicits._`, a hackish work around would be adding it to `EitherSyntax` instead.

Reply:
@kailuowang I don't believe that's possible as `try` is a reserved keyword. I've instead added `trySyntax` here https://github.com/typelevel/cats/pull/2138/files#diff-cdc99a8a4f35341b8a5aa3a52482e06fR48